docs/src/main/paradox/bootstrap/details.md

```diff
@@ -65,10 +65,16 @@ The illustration below may be of help in visualising this process:
 
 ![project structure](../images/bootstrap-forming-cluster.png)
 
-Additionally, a setting is provided, `akka.management.cluster.bootstrap.form-new-cluster` that can be disabled to only allow the
-node to join existing clusters (Case 2 below). This can be used to provide additional safety during the typical deployment
-as it is relatively rare that no cluster exists. It can be specified in your `application.conf` or provided as an
-argument to the process itself via `-Dakka.management.cluster.bootstrap.form-new-cluster=<true|false>`. By default, it is enabled.
+Cluster Bootstrap will always attempt to join an existing cluster if possible. However if no contact points are found during service discovery, a
+new cluster will be formed by default. A setting is provided, `akka.management.cluster.bootstrap.new-cluster-enabled` that can be disabled to only allow the
+node to join existing clusters (Case 2 below). This can be used to provide additional safety during the typical deployment, even when scaling out an initial deployment, as it is relatively rare that no cluster exists. 
+
+* On initial deployment use the default `akka.management.cluster.bootstrap.new-cluster-enabled=on`
```

Comment:
One question is if we should even use `off` as default? The drawback is that it will be a rather awkward first experience for those that are not reading the documentation carefully.

Reply:
@patriknw I went back and forth with that and tried both. I don't recall why but defaulting to `off` was tricky.

Reply:
I agree, lets keep the default to `on` and consider that as a separate change that needs more thinking